packages/manager/modules/pci/src/projects/project/storages/object-storage/objects/object/addUser/addUser.routing.js

```diff
@@ -0,0 +1,52 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.storages.containers.objects.addUser',
+    {
+      url: '/addUser?containerId&isHighPerfStorage',
```

Comment:
As mentioned in another PR, user shouldn't have to worry about `highPerfStorage` when accessing to a url

Reply:
Yes will remove this